pkg/cmd/run/view/view.go

```diff
@@ -355,16 +328,24 @@ func runView(opts *ViewOptions) error {
 		}
 
 		fmt.Fprintln(out)
-		fmt.Fprintln(out, "For more information about a job, try: gh run view --job=<job-id>")
-		// TODO note about run view --log when that exists
-		fmt.Fprintf(out, cs.Gray("view this run on GitHub: %s\n"), run.URL)
+		if shared.IsFailureState(run.Conclusion) {
+			fmt.Fprintf(out, "To see what failed, try: gh run view %d --log-failed\n", run.ID)
+		} else {
+			fmt.Fprintln(out, "For more information about a job, try: gh run view --job=<job-id>")
+		}
+		fmt.Fprintf(out, cs.Gray("View this run on GitHub: %s\n"), run.URL)
+
 		if opts.ExitStatus && shared.IsFailureState(run.Conclusion) {
 			return cmdutil.SilentError
 		}
 	} else {
 		fmt.Fprintln(out)
-		fmt.Fprintf(out, "To see the full job log, try: gh run view --log --job=%d\n", selectedJob.ID)
-		fmt.Fprintf(out, cs.Gray("view this run on GitHub: %s\n"), run.URL)
+		if shared.IsFailureState(selectedJob.Conclusion) {
+			fmt.Fprintf(out, "To see the log of steps that failed, try: gh run view --log-failed --job=%d\n", selectedJob.ID)
```

Comment:
Could use a second opinion on this added text.

Reply:
I like "To see the logs for failed steps, try:"